ci-conda.Dockerfile

```
@@ -24,115 +24,122 @@ ENV SCCACHE_IDLE_TIMEOUT=32768
-              cuda-cudart-dev-${PKG_CUDA_VER} \
-              cuda-cupti-dev-${PKG_CUDA_VER} \
-            # ignore the build-essential package since it installs dependencies like gcc/g++
-            # we don't need them since we use conda compilers, so this keeps our images smaller
-            && apt-get download cuda-nvcc-${PKG_CUDA_VER} \
-            && dpkg -i --ignore-depends="build-essential" ./cuda-nvcc-*.deb \
-            && rm ./cuda-nvcc-*.deb \
-            # apt will not work correctly if it thinks it needs the build-essential dependency
-            # so we patch it out with a sed command
-            && sed -i 's/, build-essential//g' /var/lib/dpkg/status \
-            && rm -rf "/var/lib/apt/lists/*"; \
-            ;; \
-          "centos"* | "rockylinux"*) \
-            yum -y update \
-            && yum -y install --setopt=install_weak_deps=False \
-              cuda-cudart-devel-${PKG_CUDA_VER} \
-              cuda-driver-devel-${PKG_CUDA_VER} \
-              cuda-gdb-${PKG_CUDA_VER} \
-              cuda-cupti-${PKG_CUDA_VER} \
-            && rpm -Uvh --nodeps $(repoquery --location cuda-nvcc-${PKG_CUDA_VER}) \
-            && yum clean all; \
-            ;; \
-          *) \
-            echo "Unsupported LINUX_VER: ${LINUX_VER}" && exit 1; \
-            ;; \
-        esac \
-        ;; \
-      *) \
-        echo "Skipping CUDA Toolkit installation for CUDA ${CUDA_VER}"; \
-        ;; \
+RUN <<EOF
+case "${CUDA_VER}" in
+  "11"*) 
+    PKG_CUDA_VER="$(echo ${CUDA_VER} | cut -d '.' -f1,2 | tr '.' '-')"
+    echo "Attempting to install CUDA Toolkit ${PKG_CUDA_VER}"
+    case "${LINUX_VER}" in
+      "ubuntu"*) 
+        apt-get update
+        apt-get upgrade -y
+        apt-get install -y --no-install-recommends \
+          cuda-gdb-${PKG_CUDA_VER} \
+          cuda-cudart-dev-${PKG_CUDA_VER} \
+          cuda-cupti-dev-${PKG_CUDA_VER}
+        # ignore the build-essential package since it installs dependencies like gcc/g++
+        # we don't need them since we use conda compilers, so this keeps our images smaller
+        apt-get download cuda-nvcc-${PKG_CUDA_VER}
+        dpkg -i --ignore-depends="build-essential" ./cuda-nvcc-*.deb
+        rm ./cuda-nvcc-*.deb
+        # apt will not work correctly if it thinks it needs the build-essential dependency
+        # so we patch it out with a sed command
+        sed -i 's/, build-essential//g' /var/lib/dpkg/status
+        rm -rf "/var/lib/apt/lists/*"
+        ;;
+      "centos"* | "rockylinux"*) 
+        yum -y update
+        yum -y install --setopt=install_weak_deps=False \
+          cuda-cudart-devel-${PKG_CUDA_VER} \
+          cuda-driver-devel-${PKG_CUDA_VER} \
+          cuda-gdb-${PKG_CUDA_VER} \
+          cuda-cupti-${PKG_CUDA_VER}
+        rpm -Uvh --nodeps $(repoquery --location cuda-nvcc-${PKG_CUDA_VER})
+        yum clean all
+        ;;
+      *) 
+        echo "Unsupported LINUX_VER: ${LINUX_VER}" && exit 1
```

```suggestion
        echo "Unsupported LINUX_VER: ${LINUX_VER}"
        exit 1
```

Let's remove the `&&` since we're no longer using it anywhere else.